packages/taquito-michelson-encoder/src/schema/storage.ts

```diff
@@ -236,27 +249,29 @@ export class Schema {
    * @param valueType type of value to look for
    *
    */
-  FindFirstInTopLevelPair<T extends MichelsonV1Expression>(storage: any, valueType: any) {
```

Comment:
Removing `<T extends MichelsonV1Expression>` seems to be a breaking change for the users. I am not sure we should make that change

Reply:
👍  ok I'll keep away form changes like these!